Source/spells.cpp

```diff
@@ -171,17 +171,21 @@ void DoResurrect(int pnum, int rid)
 		ClrPlrPath(rid);
 		plr[rid].destAction = ACTION_NONE;
 		plr[rid]._pInvincible = FALSE;
+#ifndef HELLFIRE
 		PlacePlayer(rid);
 
 		hp = 640;
 		if (plr[rid]._pMaxHPBase < 640) {
 			hp = plr[rid]._pMaxHPBase;
 		}
+#else

```

Comment:
I would prefer two hellfire ifdef's to an else.

Reply:
done